client/components/ItemActionConfirmation/UpdateMethodSelection.jsx

```diff
@@ -32,7 +33,7 @@ export const UpdateMethodSelection = ({
             </div>
         )}
 
-        { relatedPlannings.length > 0 && (
+        { relatedPlannings.length > 0 && eventsInUse.length <= 0 && (
```

Comment:
This causes a crash when attempting to cancel an Event in the series (attempted in the associated test instance)
`TypeError: Cannot read property 'length' of undefined`.

It might be better to not provide the `eventsInUse` but instead to not provide `relatedPlanning` from the `SpikeEventComponent`, which in turn will not show the related plannings warning

Reply:
That makes sense.